include/soci/type-conversion.h

```diff
@@ -346,30 +346,6 @@ class conversion_use_type<std::vector<T> >
     }
 
 private:
-    void convert_from_base()
```

Comment:
it was copy paste and have no real usage i as investigated. I hope i did correct.

Reply:
It's difficult to be sure it's not used, as it's a virtual method... it's called from `standard_use_type::post_use()` and it's not obvious to me why is this _not_ needed.

It needs to be tested with some type `T` for which `type_conversion<T>::from_base()` has a non-trivial implementation.

Reply:
Okay. I can do fix and not touch this place. 
But very strange - sodc::use bind input data. Why we need change input data? I understand than we need change output with soci::into